src/MessageBird/Common/HttpClient.php

```diff
@@ -117,10 +150,11 @@ public function performHttpRequest($method, $resourceName, $query = null, $body
         }
 
         // Some servers have outdated or incorrect certificates, Use the included CA-bundle
-        $caFile = realpath(dirname(__FILE__) . "/../ca-bundle.crt");
+        $caFile = realpath(__DIR__ . '/../ca-bundle.crt');
         if (!file_exists($caFile)) {
-            throw new Exceptions\HttpException('Unable to find CA-bundle file: ' . $caFile);
+            throw new Exceptions\HttpException(sprintf('Unable to find CA-bundle file "%s".', __DIR__ . '/../ca-bundle.crt'));
```

Comment:
if the file does not exists `realpath` returns `false` so printing the var. `$caFile` did not help